# Homework 7

#### Note: to clear the environment in R, do:
`rm(list=ls(all=TRUE))`

## **1. A potential side effect of using oral contraceptives is an increase in blood pressure. Given below are the systolic blood pressures of 10 women measured before beginning and after having taken an oral contraceptive for a 6-month period.**

> Woman | Before | After |
> :---: | :---:  | :---: | 
>   1   |  113   |  118  |
>   2   |  117   |  123  |
>   3   |  111   |  114  |
>   4   |  107   |  115  |
>   5   |  115   |  122  |
>   6   |  134   |  140  |
>   7   |  121   |  120  |
>   8   |  108   |  105  |
>   9   |  106   |  111  |
>   10  |  125   |  129  |

### **Assuming systolic blood pressure to be normally distributed, do these data suggest that using an oral contraceptive significantly increases systolic pressure?**

Step 1: Set the hypothesis
$
H_0:\mu = 0
\\
H_a:\mu >0
\\
\\
\alpha =0.05
$
\
Step 2
\
Calculate the mean, standard deviation, and standard error of the difference between the two values
\

In [1]:
x1<-c(113,117,111,107,115,134,121,108,106,125)
x2<-c(118,123,114,115,122,140,120,105,111,129)
d<-c(x1 - x2)
mean(d)
sd(d)
sd(d)/sqrt(length(d))

[1] -4

[1] 3.496029

[1] 1.105542

Step 3
\
Calculate the test statistics
$
t=\frac{\bar{d}}{\frac{s}{\sqrt{n}}}
\\
t=\frac{-4}{\frac{3.49603}{\sqrt{10}}}
\\
t=-3.62
$
\
Now we need to find the P-value using R

In [0]:
pt(-3.62,9)

Since the p-value is smaller than $\alpha$=0.05
\
We reject the null hypothesis and we accept the alternative hypothesis which means that there is a significant difference between the two means

## **2. Suppose that you mistakenly believed that the data sets from (1) were independent samples taken from two different populations: (i) women not taking contraceptives and (ii) women taking contraceptives.**

### **a. Which test would now be appropriate for examining whether there's a significant difference between the two populations?**

### **b. Carry this test out and determine whether you would have found that women taking contraceptives have a significantly higher systolic pressure.**

### **c. Explain the cause of any differences in your results when compared to the previous problem.**


# Part A

The easiest test to determine whether there is a significant differences between the two populations is an F-test
# Part B
$
\bar{X_{before}}=115.7
\\
\bar{X_{while}}=119.7
\\
S_{1}^2=78.9
\\
S_{2}^2=96.01
$
\
The first step is to set the hypothesis for the f-test since we are testing if the difference between the two variances come from the same population
\
$
H_0: \sigma_1 ^2 = \sigma_2 ^2
\\
H_a:  \sigma_1 ^2 \ne \sigma_2 ^2
$
\
F-test statistics
\
$
F=\frac{s_1 ^2}{s_2 ^2}
\\
F=\frac{78.90}{96.01}
\\
F=0.82
$
\
Now we use R to find the critical value for the f test
\
The critical value for the f-test is between 4.03 and 0.2483
\
Since 0.82 is between this two values it means that we fail to reject the null hypothesis and therefore we can say that there is no significant difference between the two population variances
\
In other words, the population variances for the two sample populations come from the same population
\
Now that we know that the samples come from the same population we can use t-statistics
\
Before we do that, we have to do set the hypothesis for this test
\
$
H_0: \mu_1 = \mu_2
\\
H_a: \mu_1 < \mu_2
$
\
For this question we have to find pooled standard deviation since we the two sample come from the same population
\
$
S_p=\sqrt{\frac{(n_1-1)s_1 ^2 + (n_2-1)s_2 ^2}{n_1 + n_2 - 2}}
\\
S_p={\frac{(10-1)*78.90 ^2 + (10-1)* 96.01 ^2}{10 + 10 - 2}}
\\
S_p=87.455
$
\
Now we use the t-test for two samples
\
$
t=\frac{(\bar{x_1} - \bar{x_2})-(\mu_1 - \mu_2)}{s_p * \sqrt{\frac{1}{n_1}+\frac{1}{n_2}}}
\\
t=\frac{(115.7 - 119.7)-(0)}{87.455 * \sqrt{\frac{1}{10}+\frac{1}{10}}}
\\
t=-0.10
$
\
Now we use R to find the p-value given the t statistics
\
The p-value is 0.46072 which is way above the alpha of 0.05
\
Therefore we fail to reject the null hypothesis and there is no difference between the two means
# Part C
In problem 8, we said that there is a significant difference between the two means while in this test we said that there was no significant difference. This is due to problem 8 assuming they are independent events which makes the probability to be more probable since it doesn't account for the intersection between the two populations like in problem 9

In [9]:
x1<-c(113,117,111,107,115,134,121,108,106,125)
x2<-c(118,123,114,115,122,140,120,105,111,129)
mean(x1)
var(x1)
mean(x2)
var(x2)
qf(0.05/2,9,9, lower.tail = FALSE)
qf(0.05/2,9,9, lower.tail = TRUE)
pt(-0.10,18)

[1] 115.7

[1] 78.9

[1] 119.7

[1] 96.01111

[1] 4.025994

[1] 0.2483859

[1] 0.4607248

## **3. A biochemist studying the activity of the enzyme triose phosphate isomerase (TPI) in a species of Drosophila wants to know if the mean activity of this enzyme is the same at $pH = 5$ and $pH = 8$.**

> Enzyme activity ($\frac{\mu M}{min}$) | | | | | |
> :---: | :---:  | :---: | :---: | :---:  | :---: 
> $\textbf{pH = 5}$ | 11.1 | 10.0 | 13.3 | 10.5 | 11.3
> $\textbf{pH = 8}$ | 12.0 | 15.3 | 15.1 | 15.0 | 13.2

### **a. Given the above data and assuming enzyme activities are normally distributed, determine if the mean activities of TPI are significantly different at these two pH levels.**

### **b. Give the 95% confidence interval for $\mu_5 - \mu_8$.**

a)
\
Step 1: Determine the hypothesis that we are going to be testing
\
$
H_0:\mu_1 - \mu_2 \geq 0
\\
H_a:\mu_1 - \mu_2 \leq 0
$
\
\
Step 2: Determine which test you are going to use
\
We are going to do an independent sample t-test
\
Ph= 5 Sample
\
$
\bar{x_1}= 11.24
\\
s_1=1.26
\\
n_1=5
$
\
\
Ph= 8 Sample
\
$
\bar{x_2}= 14.12
\\
s_2=1.45
\\
n_2=5
$
\
\
Significance level= $\alpha=0.05$
\
Standard Error
\
$\sigma_{\bar{x_1} - \bar{x_2}}=\sqrt{\frac{s_1^2}{n_1}+ \frac{s_2^2}{n_2}}$
\
$\sigma_{\bar{x_1} - \bar{x_2}}=\sqrt{\frac{1.26^2}{5}+ \frac{1.45^2}{5}}$
\
$\sigma_{\bar{x_1} - \bar{x_2}}=0.8591$
\
T test
\
$t_{observed}=\frac{\bar{x_1}- \bar{x_2}}{\sqrt{\frac{s_1^2}{n_1}+ \frac{s_2^2}{n_2}}}
\\
t_{observed}=\frac{11.24- 14.12}{0.8591}
\\
t_{observed}=-3.35242
$
Now we use R to find the p-value

In [0]:
x<-(11.24-14.12)/0.8590809
pt(x,4)

Now we calculate the critical values

In [0]:
qt(0.975,4, lower.tail = F )

Now we compare our results
\
$
T_{observed}<T_{critical}
\\
−3.35242 < -2.77644510519779
$
\
Therefore we reject the null hypothesis and accept the alternative hypothesis
Which means that the difference between the two means is signifcant

Part B
\
$
\bar{X_1}-\bar{X_2}\pm t_{0.05,8} (0.9624)
\\
11.24 -14.12\pm (2.306)*(0.9624)
\\
-2.88\pm 2.2193
$


## **4. A reproductive physiologist studying the effect of photoperiod on reproductive readiness in Japanese quail measured the cloacal gland widths of male quail exposed for 2 weeks to different day lengths:**

> |   | Long days | Short days 
> | :---: | :---: | :---: 
> | **Gland width (mm)** | $\overline{x}_1$ = 12 | $\overline{x}_2$ = 8.5 
> | **Standard deviation (mm)** | $s_1$ = 2.5 | $s_2$ = 2.0
> | **Sample size** | $n_1$ = 16 | $n_2$ = 16

### **Long days consisted of 16 hours of light and 8 hours of dark. Short days had 8 hours of daylight and 16 hours of dark. Assuming cloacal gland widths are normally distributed, are the glands significantly wider during long days?**

Step 1
\
Determine if this two samples come from the same population using an F-test
\
$
H_0: \sigma_1 = \sigma_2
\\
H_a:  \sigma_1 \ne \sigma_2
$
\
F-test statistics
\
$
F=\frac{s_1 ^2}{s_2 ^2}
\\
F=\frac{2.5^2}{2^2}
\\
F=1.5625
$
\
Now we use R to find the critical value for the f test
\
The critical value for the f-test is between 0.362140511593323 and 2.76
\
Since 1.5625 is within the critical value we fail to reject the null hypothesis and we conclude that there is evidence for this two standard deviations to from the same population
\
Step 2
\
State hypothesis
\
$
\mu_1 = long
\\
\mu_2= short
\\
H_0:\mu_1 = \mu_2
\\
H_a:\mu_1 > \mu_2
$
\
Step 3
\
Now that we know that they two variations come from the same population, we can run a t-test
\
$
t_{observed}=\frac{|\bar{x_1} - \bar{x_2}|}{\sqrt{\frac{s_1 ^2 * (n_1 -1) + s_2 ^2 (n_2 -1)}{n_1 + n_2 -2}}} * \sqrt{\frac{n_1 * n_2}{n_1 +n_2}}
\\
x_1= 12
\\
x_2= 8.5
\\
s_1=2.5
\\
s_2=2.0
\\
n_1=16
\\
n_2=16
\\
t_{observed}=\frac{|\bar{12} - \bar{8.5}|}{\sqrt{\frac{2.5 ^2 * (16 -1) + 2 ^2 (16 -1)}{16 + 16 -2}}} * \sqrt{\frac{16 * 16}{16 + 16}}
\\
t_{observed}=\frac{3.5}{\sqrt{5.125}} * \sqrt{8}
\\
t_{observed}=4.372954738
$
\
Step 3
\
Find the critical values
\
Find the t-critical value
\
$
\alpha=0.05
\\
df=n1 + n2 - 2
\\
df=16 + 16 -2
\\
df=30
$
\
$
T_{1-\alpha,30}=1.697260886593
$
Step 6
\
Compare $t_{observed}$ and $t_{critical value}$
\
$
T_{observed}>T_{critical}
\\
4.372954738> 1.697260886593
$
\
Since that is true, we reject the null hypothesis and therefore the glands are significantly wider during long days

In [26]:
n1<-16
n2<-16
qf(c(0.025,0.975),n1,n2)
qt(1-0.05,16+16-2)

[1] 0.3621405 2.7613591

[1] 1.697261

## **5. Copper sulfate is routinely used to control algae blooms in ponds and lakes. An ichthyologist believes that copper sulfate reduces the number of mucus cells certain species of fish. To test her belief, she recorded the number of mucus cells per square micron in the gill filaments of untreated fish and in fish exposed for 24 hours to copper sulfate at 1 mg/l:**

$Untreated: (16, 17, 12, 18, 11, 18, 12, 15, 16, 14, 16)$

$Exposed: (10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10)$

### **a. Assuming normality for these data, does the ichthyologist have support for her contention?**

### **b. Develop a 99% confidence interval for $\mu_{untreated} - \mu_{exposed}$ and explain in words what this calculation represents.**

# Part A
Step 1
\
Calculate the sample statistics
\
Untreated Mean=15
\
Untreated Standard Deviation=2.44
\
Exposed Mean=10
\
Exposed Standard Deviation=0
\
Step 2
\
State the hypothesis
\
$
\mu_1 = untreated
\\
\mu_2= exposed
\\
H_0:\mu_1 = \mu_2
\\
H_a:\mu_1 < \mu_2
$
\
Step 3
\
We first need to run an F-test to be able to determine if their standard deviation comes from the same population
\
The first step is to set the hypothesis for the f-test since we are testing if the difference between the two variances come from the same population
\
$
H_0: \sigma_1 = \sigma_2
\\
H_a:  \sigma_1 \ne \sigma_2
$
\
F-test statistics
\
$
F=\frac{s_1 ^2}{s_2 ^2}
\\
F=\frac{2.44}{0}
\\
F=0
$
\
Now we use R to find the critical value for the f test
\
The critical value for the f-test is between 0.29772 and 3.094589790898
\
Since 0 is not between this two values we can say that this two samples do not belong to the same population
\
Step 4
\
Knowing this we  have to run a T-test
\
$
t=\frac{|{\bar{x_1}-\bar{x_2}|}}{\sqrt{\frac{s_1 ^2}{n_1} + \frac{s_2 ^2}{n_2}}}
\\
t=\frac{|{15-10|}}{\sqrt{\frac{2.45^2}{11} + \frac{0^2}{14}}}
\\
t=\frac{|5|}{0.738702}
\\
t=6.768622021
$
\
Step 5
\
Find the t-critical value
\
$
\alpha=0.05
\\
df=n1 + n2 - 2
\\
df=11 + 14 -2
\\
df=23
$
\
T-critical=1.71387152774705
\
Step 6
\
Compare $t_{observed}$ and $t_{critical value}$
\
$
T_{observed}<T_{critical}
\\
6.76862 < 1.71387
$
\
Since that is not true, we fail to reject the null hypothesis
\
Which means that the ichthyologist have no significant data to support that
\
# Part B
$
\bar{x_1} - \bar{x_2} \pm t_{1-\frac{\alpha}{2},n_1 + n_2 - 2} * \frac{s}{\sqrt{n}}
\\
15 - 10 \pm t_{0.975,23} * \frac{2.45}{\sqrt{11}}
\\
5 \pm 1.528122706
\\
(6.528122706,3.471877294)
$
\
This is the interval at a confidence interval of 99% of the difference between the two samples

In [22]:
#Part A
untreated<-c(16, 17, 12, 18, 11, 18, 12, 15, 16, 14, 16)
mean(untreated)
sd(untreated)
exposed<-c(10,10,10,10,10,10,10,10,10,10,10,10,10,10)
mean(exposed)
sd(exposed)
n1<-length(untreated)
length(untreated)
n2<-length(exposed)
length(exposed)
qf(c(0.025,0.975),n1,n2)
qt(1-0.05,23)
qt(1-(0.05/2),23)

[1] 15

[1] 2.44949

[1] 10

[1] 0

[1] 11

[1] 14

[1] 0.2977245 3.0945898

[1] 1.713872

[1] 2.068658